src/app/courses/components/courses-page/courses-page.component.less

```diff
@@ -0,0 +1,15 @@
+:host{
+  padding: 10px 40px;
+  display: flex;
+  flex-direction: column;
+  .header{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+  .loadMore{
+    display: flex;
+    width: 200px;
+    margin: auto;
+  }
```

Comment:
This isn't an error per se, but I would recommend to avoid putting everything inside :host { }  block. It adds one level of nesting which is generally not good. And it doesn't serve any practical purpose, because styles are scoped to the component host anyway... So, as a rule of thumb I'd place all css rules that are not related to host element separately.